absl/strings/escaping_test.cc

```diff
@@ -278,6 +278,98 @@ TEST_F(CUnescapeTest, UnescapesMultipleUnicodeNulls) {
                    "\0", 5), result_string_);
 }
 
+TEST(CUnescape, NonNullTerminatedOctal1) {
+  absl::string_view reference("\\111", 2);
+
+  std::string unescaped_str;
```

Comment:
I'm probably being too pedantic, but let's make these tests more consistent with the tests above.  If we don't, someone is likely to wonder why they're different when there isn't really a reason.

The CUnescape test framework has a `result_string_` member variable.  Let's reuse that here.  We'll have to change `TEST` to `TEST_F` to make that work.

We can also rename `reference` to `original_string` to match the above tests.

Reply:
Cool, thanks for the tip. Don't worry about being pedantic, that's how good code is done. :)

I'll fix that.